src/personen/personen.lambda.ts

```diff
@@ -0,0 +1,111 @@
+import * as https from 'https';
+import { AWS } from '@gemeentenijmegen/utils';
+import { DynamoDB } from 'aws-sdk';
+import nodefetch from 'node-fetch';
+
+export async function handler (event: any, _context: any):Promise<any> {
+  const request = JSON.parse(event.body);
+  const apiKey = event.requestContext.identity.apiKey;
+
+  const idTable = new DynamoDB.DocumentClient();
+
+  const validProfile = validateFields(request.fields, apiKey, idTable);
+
+  const certKey = await AWS.getSecret(process.env.CERTIFICATE_KEY!);
+  const cert = await AWS.getSecret(process.env.CERTIFICATE!);
+  const certCa = await AWS.getSecret(process.env.CERTIFICATE_CA!);
+  const endpoint = await AWS.getParameter(process.env.LAYER7_ENDPOINT!);
+  const brpApiKey = await AWS.getSecret(process.env.BRP_API_KEY_ARN!);
```

Comment:
Deze buiten de handler-code initializeren voor we het in gebruik nemen (noemde je vanochtend denk ik al). Voor nu is een snelle optimalisatie waardoor ze niet allemaal op elkaar wachten:
```suggestion 
  const [certKey, cert, certCa, endPoint, brpApiKey] = Promise.all([
    AWS.getSecret(process.env.CERTIFICATE_KEY!),
    AWS.getSecret(process.env.CERTIFICATE!),
    AWS.getSecret(process.env.CERTIFICATE_CA!),
    AWS.getParameter(process.env.LAYER7_ENDPOINT!),
    AWS.getSecret(process.env.BRP_API_KEY_ARN!)
  ]);
```